drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/InsertColumnCommand.java

```diff
@@ -34,68 +32,42 @@
  * <code>Command</code> to <b>insert</b> a column.
  */
 @Dependent
-public class InsertColumnCommand extends AbstractCommand {
-
-    private ScenarioGridModel model;
-    private String columnId;
-    private int columnIndex;
-    private boolean isRight;
-    private boolean asProperty;
-
-    public InsertColumnCommand() {
-    }
-
-    /**
-     * @param model
-     * @param columnId
-     * @param columnIndex
-     * @param isRight when <code>true</code>, column will be inserted to the right of the given index (i.e. at position columnIndex +1), otherwise to the left (i.e. at position columnIndex)
-     * @param asProperty when <code>true</code>, column will use the <b>instance</b> header of the original one, so to create a new "property" header under the same instance
```

Comment:
Please preserve these comments (maybe you can move them to `ScenarioSimulationContext`?)